src/hooks/useId.js

```diff
@@ -1,14 +1,20 @@
+import { useState } from 'react'
 import yelp from '../api/yelp'
 
-export default async function idApi(id, actionOne, actionTwo) {
-  try {
-    const response = await yelp.get(`${id}`)
-    actionOne(response.data)
-  } catch (err) {
-    actionTwo(
-    'We are so sorry, but there is no answer from the server, please try again later!'
-    )
+
+export default () => { 
+  const [result, setResult] = useState(null)
+  const [errorMsg, setErrorMsg] = useState(false)
+
+  async function idApi(id) {
+    try {
+      const response = await yelp.get(`${id}`)
+      setResult(response.data)
+      setErrorMsg(false)
+    } catch (err) {
+      setErrorMsg(true)
+    }
```

Comment:
This change was made earlier, about one month before. Just was not committed to the main brunch.